Command/IndexRebuildCommand.php

```diff
@@ -39,7 +45,28 @@ public function configure()
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-        $eventDispatcher = $this->getContainer()->get('event_dispatcher');
+        $formatterHelper = new FormatterHelper();
+        $questionHelper = new QuestionHelper();
+
+        $container = $this->getContainer();
+        $startTime = microtime(true);
+        $eventDispatcher = $container->get('event_dispatcher');
+        $resumeManager = $container->get('massive_search.reindex.resume_manager');
+        $env = $container->getParameter('kernel.environment');
+
+        if ($env !== 'prod') {
```

Comment:
Would also add an option to disable this warning, so that people using this command and being aware of that don't have their output cluttered.